### PR TITLE
iptables 1.8.9

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

---

relates to #127086

### CODE REVIEW

File: Formula/iptables.rb [hunk 1]
```diff
@@ -33,6 +32,6 @@ def install
   end
 
   test do
-    assert_match "Permission denied (you must be root)", shell_output("iptables-nft --list-rules 2>&1", 4)
+    assert_match "Permission denied (you must be root)", shell_output("#{bin}/iptables-nft --list-rules 2>&1", 4)
```
Comment (cho-m) [inline Formula/iptables.rb:35]:
```suggestion
    assert_match "Permission denied (you must be root)", shell_output("#{sbin}/iptables-nft --list-rules 2>&1", 4)
```

**Comment by dawidd6** [comment]
@carlocab @MikeMcQuaid 

```
==> Using Homebrew/homebrew-test-bot 1b0ce2e (Merge pull request #890 from carlocab/dump-skipped-formulae)
==> Using Homebrew/brew 4.0.10-111-g4c4fb3b9a (Merge pull request #15101 from Homebrew/dependabot/bundler/Library/Homebrew/net-http-persistent-4.0.2)
==> Testing Homebrew/homebrew-core 82d4018c5ab (Merge caf1faf02ffba08da125b9222d3f9a5e4aa3b87d into 1b0e6747aac2120519905[17](https://github.com/Homebrew/homebrew-core/actions/runs/4573668584/jobs/8074636863?pr=127087#step:7:18)e9752584ea5d3beb0):
==> Skipped or failed formulae:
Error: Failed to find bottle for 'iptables'.
Error: Process completed with exit code 1.
```

Guess there is no fail-safe for a os-specific formulae?

**Comment by carlocab** [comment]
Yes, I guess not. Let me look at it.

**Comment by carlocab** [comment]
Hmmm. This should be showing up in `skipped_or_failed_formulae`, and then skipped. Not sure what's wrong. Looking... 👀

**Comment by carlocab** [comment]
Incidentally, log upload on macOS is now broken:
```
No files were found with the provided path: /Users/brew/actions-runner/_work/homebrew-core/homebrew-core/bottles/logs. No artifacts will be uploaded.
```

**Comment by carlocab** [comment]
> Hmmm. This should be showing up in `skipped_or_failed_formulae`, and then skipped. Not sure what's wrong. Looking... 👀

Ah, here's why. In Homebrew/homebrew-test-bot#889, we started passing the list of `skipped_or_failed_formulae` in the `bottles` artifact. However, when no bottle is built, we skip uploading the `bottles`, which contains our list of skipped or failed formulae:

https://github.com/Homebrew/homebrew-core/blob/d75c1bee788b4fd1357c2aecbe30fa77e87076f6/.github/workflows/tests.yml#L340-L342

**Comment by carlocab** [comment]
#127177

**Review by carlocab — APPROVED** [review]

**Pull request merged**